ksml/src/main/java/io/axual/ksml/python/PythonContext.java

```diff
@@ -43,19 +43,28 @@ public class PythonContext {
     @Getter
     private final DataObjectConverter converter;
 
-    public PythonContext() {
+    public PythonContext(PythonContextConfig config) {
         this.converter = new DataObjectConverter();
 
-        log.debug("Setting up new Python context");
+        log.debug("Setting up new Python context (fileAccess={}, socketAccess={}, nativeAccess={}, " +
```

Comment:
If you create a toString method in the PythonContextConfig class you can simplify this.
You can use to lombok ToString annotation and specify which fields to use